cypress_test/integration_tests/common/calc_helper.js

```diff
@@ -138,6 +138,24 @@ function removeTextSelection() {
 	cy.log('<< removeTextSelection - end');
 }
 
+// Click on rows header and select Hide rows from a context menu
+function hideSelectedRows() {
+	cy.log('>> hideSelectedRows - start');
+
+	cy.cGet('[id="test-div-row header"]')
+		.then(function(header) {
+			expect(header).to.have.lengthOf(1);
+			var rect = header[0].getBoundingClientRect();
+			var posX = (rect.right + rect.left) / 2.0;
+			var posY = (rect.top + rect.bottom) / 2.0;
+			cy.cGet('body').rightclick(posX, posY);
```

Comment:
You should be able to save the costly getBoundingClientRect
Calling rightClick on header[0] with no parameters.

Or was there a reason ?

https://docs.cypress.io/api/commands/rightclick

Reply:
copy-paste :P ok, in this case it should wokr (center), but in places where it is a specific place on canvas - not

Reply:
no it doesn't work, possibly the handler is on some other layer/thing and we just click at coordinate to propagate into the right thing... I don't know.